BuilderImage.Dockerfile

```diff
@@ -0,0 +1,46 @@
+#FROM cimg/node:16.16.0
```

Comment:
This file needs to be cleaned up. All of the additional commented out commands are unnecessary.

Reply:
Cleaned out commented out commands.